core/trino-parser/src/test/java/io/trino/sql/parser/TreeNodes.java

```diff
@@ -35,7 +35,7 @@
 
 import static java.util.Collections.emptyList;
 
-class TreeNodes
+public class TreeNodes
```

Comment:
This class is meant to be private for testing the SQL parser. Don't make it public.

Reply:
The `*Formatter` classes are in `io.trino.sql`, one level up from `parser` so I created the test in the same package.

I removed the dependency & re-impl:ed `simpleType` & `location` inside the test for now. We might end up with more duplications though if we add more formatter tests like this.